lib/seagullService.js

```diff
@@ -22,15 +22,16 @@ var restify = require('restify');
 
 var log = require('./log.js')('seagullService.js');
 
-function createServer(serverConfig, crudHandler, userApiClient, metrics) {
+function createServer(serverConfig, crudHandler, userApiClient, metrics, gatekeeperClient) {
```

Comment:
I would put the gatekeeperClient parameter before the metrics, but that's just me.  (This would apply to the other changes in this file as well.)